### DISCUSSION TITLE
What is the unit of `keyball61/design_data/*.dxf`?

### DISCUSSION THREAD

**Discussion by knttnk** [Yowkees/keyball#136] [Q&A] [answered]
I'm using [Keyball61_Rev1_BallSide_Bottom_Acryl_Clear2mm.dxf](https://github.com/Yowkees/keyball/blob/main/keyball61/design_data/Keyball61_Rev1_BallSide_Bottom_Acryl_Clear2mm.dxf) to design my hard case of the Keyball61. But its unit of length is not specified, is it?

Thank you.

**Comment by Yowkees** [comment, marked as answer]
My Adobe Illustrator said 1 unit = 1 Points.
For the record: 1 point is 0.352778 mm.

So sorry my late reply.

**Reply by knttnk** [reply]
No problem.
Thank you!
